src/sentry/static/sentry/app/components/debugFileFeature.tsx

```diff
@@ -24,7 +24,7 @@ const FEATURE_TOOLTIPS = {
 
 type Props = {
   feature: 'symtab' | 'debug' | 'unwind' | 'sources';
-  available?: true;
+  available?: boolean;
```

Comment:
I believe this type was incorrect..@matej please do you have more context about it?

Reply:
Yeah, looks like it was supposed to be a boolean 👍